src/main/java/io/openliberty/tools/common/plugins/util/ServerFeatureUtil.java

```diff
@@ -390,19 +447,48 @@ private Set<String> parseFeatureManagerNode(Element node) {
                         if (content.isEmpty()) {
                             warn("An empty feature was specified in a server configuration file. Ensure that the features are valid.");
                         } else {
-                            result.add(content);
+                        	features.add(content);
                         }
                 	}
                 }
             }
         }
-        return result;
+        NodeList platformElements = node.getElementsByTagName("platform");
+        if (platformElements != null) {
+            for (int j = 0; j < platformElements.getLength(); j++) {
+                String content = platformElements.item(j).getTextContent();
+                if (content != null) {
+                	content = content.trim();
+                	if (content.contains(":")) {
```

Comment:
Checking for `:` was for handling user features. I don't think that applies to the platform elements.